vulcanai2/models/layers.py

```diff
@@ -121,16 +127,14 @@ def __init__(self, in_features, out_features,
                                         norm, dropout)
         self.in_features = in_features
         self.out_features = out_features
-
+        self.activation = activation
```

Comment:
why store it within the class attributes? I don't believe it is used outside of the init for Dense or Conv Unit

Reply:
That was something I added during testing and forgot to remove.